### PR TITLE
PlaySongTest

### PR DESCRIPTION
**Pull request by marianay77**

### CODE REVIEW

File: src/test/java/Homework18.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        WebElement soundBarPlay = driver.findElement (By.cssSelector ("div.wrapper"));
+       Thread.sleep (2000);
+        playButton.click();
+        Assert.assertTrue(soundBarPlay.isDisplayed ());
+
+
+    }
+
+
+
```
Comment (Berichicko) [inline src/test/java/Homework18.java:30]: **Please delete the empty line!** An Empty Line is a "Code Smell"
The short answer is, _They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce._

File: src/test/java/Homework18.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        provideEmail ("demo@class.com");
+        providePassword ("te$t$tudent");
+        clickSubmit ();
+        Thread.sleep (2000);
+    }
+
+
+    public void playButton () throws InterruptedException {
+        WebElement playButton = driver.findElement (By.cssSelector (".album-thumb-wrapper .fa.fa-play"));
+        WebElement soundBarPlay = driver.findElement (By.cssSelector ("div.wrapper"));
```
Comment (Berichicko) [inline src/test/java/Homework18.java:21]: How can I beautify code in IntelliJ:

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

image

File: src/test/java/Homework18.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+public class Homework18 extends BaseTest{
+
+    @Test
+    public void playSong () throws InterruptedException {
+
+        navigateToPage ();
+        provideEmail ("demo@class.com");
+        providePassword ("te$t$tudent");
+        clickSubmit ();
+        Thread.sleep (2000);
```
Comment (Berichicko) [inline src/test/java/Homework18.java:15]: **Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
    
For example

```
 @Test

    public void testChangeProfileName (){
        // GIVEN

        String newName  = getRandomString();

        String currentPassword = "te$t$tudent";

        provideEmail("demo@class.com");

        providePassword();

        clickSubmitBtn();

 

        // WHEN

        openUserProfilePage();

        setName(newName);

        setCurrentPassword(currentPassword);

 

 

        // THEN

        Assert.assertTrue(getSuccessPopUp().isDisplayed());

}
```

One test - One Assert

For example

File: src/test/java/Homework18.java (hunk 1 above)
Comment (marianay77) [reply src/test/java/Homework18.java:30]: done

File: src/test/java/Homework18.java (hunk 2 above)
Comment (marianay77) [reply src/test/java/Homework18.java:21]: i tried but it didn't

**Pull request merged**